tensorflow/lite/tools/optimize/operator_property.cc

```diff
@@ -996,6 +996,11 @@ OperatorProperty GetOperatorProperty(OpVariant op_variant) {
       property.restrict_same_input_output_scale = true;
       property.version = 2;
       break;
+    case BuiltinOperator_WHERE:
+      property.inputs = {{0, {}}};
+      property.outputs = {{0, {}}};
+      property.version = 1;
```

Comment:
Version not incremented as the kernel and input/output types remain unchanged (where takes BOOL input and outputs INT64 whether it is quantized or not). Let me know if the version needs to be incremented for supporting quantization anyway.